Makefile

```diff
@@ -4,59 +4,82 @@ add_targets:
 	@echo "add_targets"
 	rustup target add aarch64-apple-ios aarch64-apple-ios-sim x86_64-apple-ios
 
-build:
+
+compile:
 	@echo "build aarch64-apple-ios"
-	@cargo build --release --target aarch64-apple-ios
+	@# @cargo build --release --target aarch64-apple-ios
+	@cargo build --target aarch64-apple-ios
 	@echo "lipo"
-	@lipo -create \
+	@# @lipo -create \
 		-output target/lib$(TARGET)-ios.a \
 		target/aarch64-apple-ios/release/lib$(TARGET).a
+	@lipo -create \
+		-output target/lib$(TARGET)-ios.a \
+		target/aarch64-apple-ios/debug/lib$(TARGET).a
 
 	@echo "build aarch64-apple-ios-sim"
-	@cargo build --release --target aarch64-apple-ios-sim
+	@# @cargo build --release --target aarch64-apple-ios-sim
+	@cargo build --target aarch64-apple-ios-sim
 
 	@echo "build x86_64-apple-ios"
-	@cargo build --release --target x86_64-apple-ios
+	@# @cargo build --release --target x86_64-apple-ios
+	@cargo build --target x86_64-apple-ios
 
-	@lipo -create \
+	@# @lipo -create \
 		-output target/lib$(TARGET)-sim.a \
```

Comment:
I would probably just remove these lines if they aren't used anymore

Reply:
I had commented them because we can switch between **release and debug builds** by commenting/uncommenting, I see it is not clean for now as we could have added few variables for the debug/release binary path.

we can clean it up later.